targets/TARGET_WICED/wiced_port/WICED/RTOS/mbed/WWD/wwd_rtos.cpp

```diff
@@ -0,0 +1,304 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "wwd_rtos.h"
+#include "wwd_constants.h"
+#include "wwd_assert.h"
+#include "wwd_rtos_interface.h"
+#include "wiced_utilities.h"
+#include "mbed.h"
+#include "ticker_api.h"
+
+
+/**
```

Comment:
are these docs needed here? Not done in the header file? If they are, this is duplicate and pollutes code file

Reply:
They're copied from the header file, but the header file is hard to track down. I can remove them if we'd like.